hpycc/get.py

```diff
@@ -275,12 +276,12 @@ def get_thor_file(connection, thor_file, max_workers=15, chunk_size=10_000,
         Data type for data or columns. E.g. {‘a’: np.float64, ‘b’:
         np.int32}. If converters are specified, they will be applied
         INSTEAD of dtype conversion. If a dict is given, any
-        unspecified columns will be returned as str. Nine by default.
+        unspecified columns will be returned as str. None by default.
```

Comment:
Look into returning the schema type instead of string when dtype is set to None. Also include this for when the columns are also not specified.